test/offline/storage_unit.js

```diff
@@ -869,6 +869,43 @@ describe('Storage', function() {
         }).catch(fail).then(done);
       });
     });  // describe('default track selection callback')
+
+    describe('temporary license', function(){
+
+      beforeEach(function(){
+        storage.configure({ isPersistentLicense: false });
+      });
+
+      it('stores basic manifests', function(done) {
+        var originalUri = 'fake://foobar';
+        storage.store(originalUri)
+            .then(function(data) {
+              expect(data).toBeTruthy();
+              // Since we are using a memory DB, it will always be the first one.
+              expect(data.offlineUri).toBe('offline:0');
+              expect(data.originalManifestUri).toBe(originalUri);
+              expect(data.duration).toBe(0);  // There are no segments.
+              expect(data.size).toEqual(0);
+              expect(data.tracks).toEqual(tracks);
+            })
```

Comment:
Kind of.  I was thinking of putting checks for `drmInfo` into the offline sessions test below.  Otherwise, running that test in isolation could result in a pass even if it was clear content.

Since the feature under test is that we do temporary licenses instead of persistent licenses, I think all three of these could be one test.  In my opinion, the test above ("stores basic manifest") is irrelevant, and these other two should be combined into "does not store offline sessions").